lisa/tools/df.py

```diff
@@ -54,3 +55,21 @@ def get_partition_by_mountpoint(
             if partition.mountpoint == mountpoint:
                 return partition
         return None
+
+    def get_filesystem_available_space(
+        self, path: str = ".", force_run: bool = False
+    ) -> float:
+        # return the input 'path' available space in GB by running df
+        # If 'path' doesn't exist, return 0
+        # The df cmd output is in the following format:
+        # <Filesystem> <1K-blocks> <Used> <Available> <Use%> <Mounted on>
+        # Note : <Available> are in 1-k blocks, not in bytes
+        available_space = 0.0
+        out = self.run(parameters=path, sudo=True, force_run=force_run).stdout
+
+        df_entries = find_patterns_groups_in_lines(out, [self._DF_ENTRY_REGEX])[0]
+        if len(df_entries) == 0:  # path not exist
```

Comment:
Add debug level log to explain the path is not found, returned 0 available space.